### PR TITLE
feat(cmd): allow setting log level for start comamnd

### LINKED ISSUE #210: Logging level should be configurable via flag or config file on startup
Right now the logging level is hardcoded to `LevelInfo`, see: https://github.com/celestiaorg/celestia-node/blob/main/cmd/celestia/main.go#L29.

This should have a default that can be overridden with a flag or config file.

### PR DESCRIPTION
**Pull request by Wondertan**
Closes #210.

Btw, you can also set logging level through ENV

### CODE REVIEW

**Review by liamsi — APPROVED** [review]
utACK

File: cmd/start.go [hunk 1]
```diff
@@ -63,6 +72,11 @@ func Start(repoName string, tp node.Type) *cobra.Command {
 	}
 
 	cmd.Flags().String(cfgAddress, "", "Indicates node to connect to the given remote core node")
+	cmd.Flags().String(
+		loglevel,
+		"info",
+		"DEBUG, INFO, WARN, ERROR, DPANIC, PANIC, FATAL, and\n// their lower-case forms.",
```
Comment (liamsi) [inline cmd/start.go:78]: I'm assuming the `\n//` just creates a new-line?

Comment (Wondertan) [reply cmd/start.go:78]: Yeah

**Review by jbowen93 — APPROVED** [review]
LGTM

**Review by renaynay — APPROVED** [review]

**Pull request merged**